acceptance/tests/facts/debian.rb

```diff
@@ -83,6 +83,23 @@
     assert_match(value, fact_on(agent, fact))
   end
 
+  step "Ensure a primary networking interface was determined."
+  primary_interface = fact_on(agent, 'networking.primary')
+  assert_not_equal("", primary_interface)
+
+  step "Ensure bindings for the primary networking interface are present."
+  expected_bindings = {
+                        "networking.interfaces.#{primary_interface}.bindings.0.address" => /\d+\.\d+\.\d+\.\d+/,
+                        "networking.interfaces.#{primary_interface}.bindings.0.netmask" => /\d+\.\d+\.\d+\.\d+/,
+                        "networking.interfaces.#{primary_interface}.bindings.0.network" => /\d+\.\d+\.\d+\.\d+/,
+                        "networking.interfaces.#{primary_interface}.bindings6.0.address" => /[a-z0-9:]+/,
```

Comment:
I believe these are all hex values, so the range only needs to be `a-f` not `a-z`.